test/module/shared_model/bindings/query-test.py

```diff
@@ -77,5 +77,9 @@ def test_get_role_permissions(self):
     query = self.builder.getRolePermissions("user").build()
     self.assertTrue(self.check_proto_query(self.proto(query)))
 
+  def test_get_transactions(self):
+    query = self.builder.getTransactions([iroha.Hash("1" * 256), iroha.Hash("2" * 256)]).build()
```

Comment:
I don't know much about this code, but `"1" * 256` is 256 bytes? sha3_256 is 256 bit, so maybe `"1" * 32`?

Reply:
Yeah, you're write. Anywhere it just builder test and it doesn't really care about the hash size